[2024-01-28T16:02:16.664+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-02T00:00:00+00:00 [queued]>
[2024-01-28T16:02:16.880+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-02T00:00:00+00:00 [queued]>
[2024-01-28T16:02:16.884+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T16:02:18.145+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-03-02 00:00:00+00:00
[2024-01-28T16:02:18.188+0000] {standard_task_runner.py:60} INFO - Started process 3453 to run task
[2024-01-28T16:02:18.215+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-03-02T00:00:00+00:00', '--job-id', '1366', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpmsu8acbk']
[2024-01-28T16:02:18.223+0000] {standard_task_runner.py:88} INFO - Job 1366: Subtask get_stackoverflow_data
[2024-01-28T16:02:18.722+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-02T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T16:02:19.764+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-03-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-02T00:00:00+00:00'
[2024-01-28T16:02:22.619+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T16:02:22.668+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230302T000000, start_date=20240128T160216, end_date=20240128T160222
[2024-01-28T16:02:22.845+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T16:02:23.019+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
